Utilizr.Tests/Domain/DomainHelperTest.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using NUnit.Framework;
+using Utilizr.Util;
+
+namespace Utilizr.Tests.Domain
+{
+    [TestFixture]
+    public class DomainHelperTest
+    {
+        [Test]
+        public void SubDomainTest()
+        {
+            string fullUrl = "aaa.bbb.ccc.com";
+            var subdomain = DomainHelper.GetSubDomain(fullUrl);
+            Assert.That(subdomain, Is.EqualTo("ccc.com"));
+
+            fullUrl = "ccc.com";
+            subdomain = DomainHelper.GetSubDomain(fullUrl);
+            Assert.That(subdomain, Is.EqualTo(null));
+        }
+
+        [Test]
+        public void DomainTest()
+        {
+            string fullUrl = "aaa.bbb.ccc.com";
+            var domain = DomainHelper.GetDomain(fullUrl);
+            Assert.That(domain, Is.EqualTo("aaa.bbb.ccc.com"));
+
+            fullUrl = "www.bbb.ccc.com/xxx/www";
+            domain = DomainHelper.GetDomain(fullUrl);
+            Assert.That(domain, Is.EqualTo("www.bbb.ccc.com"));
+        }
+    }
```

Comment:
We can use the `TestCase` attributes here to improve coverage and reduce duplication in the tests themselves.

If we have something like `string testUrl` and `string expectedResult` parameters, we can then do something like:

```
[TestCase("http://test.com", "test.com")]
[TestCase("http://test.com?param=something", "test.com")]
```

Not an exhaustive lists, it would be good to have all sorts of things here.